components/related_news_component.py

```
@@ -0,0 +1,29 @@
+    세션 상태의 'generated_articles' 키에서 기사 목록을 가져와서
+    첫 번째 기사의 제목과 카테고리를 사용하여 관련 기사를 검색하고 표시합니다.
+    """
+
+    articles = st.session_state.get("generated_articles", [])
+
+    if not articles:
+        st.info("아직 저장된 기사가 없습니다.")
+        return
+
+    # print("ARTICLES : ", articles)
+    idx = len(articles) - 1;
```

Ruff (0.11.9)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)